cpe_workspaceone/resources/cpe_workspaceone.rb

```diff
@@ -21,6 +21,7 @@
   manage if manage?
   install if install?
   enforce_mdm_profiles if enforce_mdm_profiles?
+  manage_cli_config if manage_cli_config?
```

Comment:
do you think this should be before `enforce_mdm_profiles`?

Reply:
Yes, that makes most sense - done